src/stores/chat.ts

```diff
@@ -298,11 +384,9 @@ export const useChatStore = defineStore('chat', () => {
 		const deleteIndex = chatBlocks[token].findIndex((block) => Math.max(...block) < idToDelete)
 		if (deleteIndex === -1) {
 			// Not found, nothing to delete
-			return
 		} else if (deleteIndex === 0) {
 			// If first block is to be deleted, remove all blocks
-			purgeChatStore(token)
-			return
+			delete chatBlocks[token]
 		} else {
```

Comment:
```suggestion
		} else if (deleteIndex !== -1) {
```
and remove the first condition